test/fhirtypes/ElementDefinition.setInstancePropertyByPath.test.ts

```diff
@@ -110,5 +110,35 @@ describe('ElementDefinition', () => {
       expect(status.code[0]).toEqual({ code: 'foo', system: 'http://example.com' });
       expect(status.code[1]).toEqual({ code: 'bar', system: 'http://example.com' });
     });
+
+    // Children of primitives
+    it('should set a child of a primitive instance property which has a value', () => {
+      status.setInstancePropertyByPath('short', 'foo', resolve);
+      status.setInstancePropertyByPath('short.id', 'bar', resolve);
+      expect(status.short).toBe('foo');
+      // @ts-ignore
+      expect(status._short.id).toBe('bar');
```

Comment:
Do you think we should create new `_` properties in `StructureDefinition`/`ElementDefinition` for all primitive properties?  Then we could avoid the `@ts-ignore`?

Reply:
I think for now, I am going to vote no. I feel like the number of `_` elements that we would add could kinda muddy up what is right now a pretty clear representation of what can be in an ED or SD. If we find that we are referencing these `_` elements a lot in other parts of the code, and are using a lot of `@ts-ignore`, I'd be for it. But I'd prefer to push it off until we feel like we have to, unless you feel really strongly.

Reply:
OK. I'm fine with that.  Just thought I'd ask!